lib/classes/PluginManager.js

```diff
@@ -148,7 +152,27 @@ class PluginManager {
     if (pluginsObject.localPath) {
       module.paths.unshift(pluginsObject.localPath);
     }
-    this.loadPlugins(pluginsObject.modules);
+    this.loadPlugins(pluginsObject.modules
+      .filter(name => name !== '@serverless/enterprise-plugin'));
+  }
+
+  loadEnterprisePlugin() {
+    const qualifiedEnterpriseModulePath = path.join(
+      __dirname, '../../node_modules/@serverless/enterprise-plugin');
+    // eslint-disable-next-line global-require
+    const sfePkgJson = require(`${qualifiedEnterpriseModulePath}/package.json`);
+    if (config.getGlobalConfig().enterpriseDisabled ||
+        !semver.satisfies(process.version, sfePkgJson.engines.node)) {
+      return;
+    }
+    this.loadPlugins([qualifiedEnterpriseModulePath]);
+    if (this.serverless.enterpriseEnabled) {
+      const updates = updateNotifier({ pkg: sfePkgJson, interval: 1 });
+      if (updates.update) {
+        this.serverless.cli.log('An updated version of Serverless Enterprise is available. ' +
+                                'Please upgrade by running `npm i -g serverless`');
```

Comment:
AFAIK This won't force an update of SFE, as it's a dependency of SFO, where in global installation we lock dependencies updates with `package-lock.json`.

With a local installation it'll probably be possible (as then `package-lock.json` is ignored), but still I think `npm install` is not that reliable mean for that, and that could only be forced via `npm update`.

It seems that in this relation model (where SFE is a SFO dependency) the only reliable way to force SFE upgrades, is via new SFO releases.

Reply:
AFAIK there is no lock file for global install. it worked when i tested this. docs?

Reply:
We somehow publish SFO with `package-lock.json` but indeed npm doesn't take it into account in global install.

Still there's no guarantee npm will upgrade the sub dependencies, it works reliably on same machine on which you publish new version of sub dep (it lands in same npm cache), but when you try to confirm it on some other machine, it's not picked.

As I tested:
- `npm install -g serverless`, doesn't guarantee update neither of `serverless` nor of `@serverless/enterprise-plugin` dependency
- `npm update -g serverless`, guaratnees upgrade of `serverless` but doesn't gurantee update of `@serverless/enterprise-plugin` dependency

Still it might be that it happens in reasonable time (I didn't test that), but for sure it doesn't work right after new version is published

Reply:
@medikoo, since I had to publish a bug fix on the SFE plugin (`v1.0.1`) I was able to test this. It updated the plugin as I expected when I updated serverless.

Reply:
Ok, I've also checked that `npm update -g serverless` will do nothing if there's no new version of`serverless`, while `npm install -g serverless` attempts to refresh it so it's still a safer choice